Repository/PhpcrOdmRepository.php

```diff
@@ -116,4 +119,20 @@ protected function buildCollection(array $documents)
 
         return $collection;
     }
+
+    /**
+     * Returns all versions of a resource.
+     *
+     * @param string $path The path to the resource.
+     *
+     * @return VersionList The versions stored for this path.
+     *
+     * @throws NoVersionFoundException  If no version can be found.
+     * @throws InvalidArgumentException If the path is invalid. The path must
+     *                                  be a non-empty string starting with "/".
+     */
+    public function getVersions($path)
+    {
+        return new VersionList($path, []);
```

Comment:
it was just a quick fix to implement it, no logic. So the test would be
like a setter-getter test ;-)
In generall we should add the phpcr versioning logic when available.

Am 19.05.2016 um 13:28 schrieb dantleech:
>
> In Repository/PhpcrOdmRepository.php
> <https://github.com/symfony-cmf/resource/pull/15#discussion_r63861958>:
>
> > @@ -116,4 +119,20 @@ protected function buildCollection(array $documents)
> >  
> >          return $collection;
> >      }
> > +
> > +    /**
> > +     * Returns all versions of a resource.
> > +     *
> > +     * @param string $path The path to the resource.
> > +     *
> > +     * @return VersionList The versions stored for this path.
> > +     *
> > +     * @throws NoVersionFoundException  If no version can be found.
> > +     * @throws InvalidArgumentException If the path is invalid. The path must
> > +     *                                  be a non-empty string starting with "/".
> > +     */
> > +    public function getVersions($path)
> > +    {
> > +        return new VersionList($path, []);
>
> Can you add a test for this in |PhpcrOdmRepositoryTest| ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/symfony-cmf/resource/pull/15/files/bb855320dfc8f8e54706c37dea6693feb52ce691#r63861958>
>